src/app/app.html

```diff
@@ -1,10 +1,12 @@
 <div>
-  <d3-data [src]="dataUrl" (data)="lineLoaded($event)" [accessor]="parseValues" [callback]="log"></d3-data>
-  <d3-data [src]="pieDataUrl" (data)="pieLoaded($event)" [accessor]="parseValues" [callback]="log"></d3-data>
+  <d3-data [src]="dataUrl" (data)="line = $event.rows" [accessor]="parseValues" [callback]="log"></d3-data>
+  <d3-data [src]="pieDataUrl" (data)="pie = $event.rows" [accessor]="parseValues" [callback]="log"></d3-data>
+  <input [value]="test" (input)="test=$event.target.value">
```

Comment:
This is me trying to test regular input binding change detection which is working. Also as an interesting side effect the charts do update somewhat when you type.